Doc/c-api/exceptions.rst

```diff
@@ -460,12 +460,45 @@ Querying the error indicator
          }
 
 
-.. c:function:: void PyErr_GetExcInfo(PyObject **ptype, PyObject **pvalue, PyObject **ptraceback)
+.. c:function:: PyObject* PyErr_GetActiveException(void)
+
+   Retrieve the active exception instance, as would be returned by :func:`sys.exception`.
+   This refers to an exception that was *already caught*, not to an exception that was
+   freshly raised. Returns a new reference to the exception or to ``Py_None``.
```

Comment:
The implementation seems to return NULL, not None, if there is no active exception.

I prefer NULL :-)

Reply:
The code is actually inconsistent about this now - it could return either NULL or Py_None, depending on what's in the thread state. I'll make it always NULL.